src/OrchardCore/OrchardCore.DisplayManagement/PositionWrapper.cs

```diff
@@ -30,7 +30,7 @@ public PositionWrapper(IHtmlContent value, string position)
 
         public PositionWrapper(string value, string position)
         {
-            _value = new HtmlString(HtmlEncoder.Default.Encode(value));
+            _value = new HtmlString(value);
```

Comment:
I need to double-check this one, I think it might still need to get encoded.

Reply:
I will check this one

Reply:
It is only used in `Shape.cs`, we use it when adding an item that is a `string` or an `IHtmlContent`. If it is a string we convert it to an `HtmlString`, because _value is an `IHtmlContent`, and that's why we were encoding the original string.

Yes, i think it needs to be encoded.

But maybe better to use here

    using Microsoft.AspNetCore.Mvc.ViewFeatures
    ...

    _value = new StringHtmlContent(value);

So that it will be encoded through the `PositionWrapper.WriteTo()` method and with the right encoder.